bagel/validate.py

```diff
@@ -0,0 +1,60 @@
+import csv
+from datetime import datetime
+
+
+def validate_headings():
+    # read the headings of the csv and confirm they match the expected list of headings
+    correct_headings = [
+        "Timestamp",
+        "Processing",
+        "Title proper",
+        "Name of part / expansion",
+        "Number of players",
+        "Game duration",
+        "Recommended age",
+        "Central Library Barcodes",
+        "Crown Heights Barcodes",
+        "Bushwick Barcodes",
+        "McKinley Park Barcodes",
+        "New Utrecht Barcodes",
+        "Windsor Terrace Barcodes",
+        "Price",
+        "Other titles",
+        "Subtitle",
+        "Authors/designers",
+        "ISBN",
+        "UPC",
+        "Place of publication",
+        "Publisher",
+        "Date of publication",
+        "Description/summary",
+        "List of components",
+        "Email Address",
+    ]
+    with open("temp/metadata.csv", "r", encoding="utf-8") as csvfile:
+        reader = csv.reader(csvfile)
+        csv_header = next(reader)
+        try:
+            assert csv_header == correct_headings
```

Comment:
Nice use of `assert` in code 👍 
Don't be afraid to use assert in this fashion! `assert` does not only belongs in tests.